### PR TITLE
Update to SDK 3.3.0

### PR DESCRIPTION
**Pull request by AlCatt91**
Minor compatibility fixes (mainly CI-related, with some annoying bugs).
Still need to update paperspace links - I don't believe 3.3 is available yet there?

### CODE REVIEW

**Review by danjust — APPROVED** [review]
LGTM. Only question might be whether you should wait for the update on Paperspace before you merge?

**Comment by AlCatt91** [comment]
Good point, yes, let's wait: everything should still be compatible with 3.2, but for the sake of CI and other PRs it's probably better to put this on hold.

**Review by danjust — APPROVED** [review]
Looks all good. I also did a quick test of the notebooks on paperspace

**Pull request merged**
